### DISCUSSION TITLE
Disabling email verification

### DISCUSSION THREAD

**Discussion by Pato05** [WeblateOrg/weblate#10996] [Q&A] [answered]
Is there any way to disable email verification in weblate? I only need it for a small project, and I don't need open registration, I can register all the people working with my by myself, therefore I don't really need email verification..
The only thing i found is the ACCOUNT_EMAIL_VERIFICATION Django option, but I'm not really sure that it is what I need.

**Comment by nijel** [comment, marked as answer]
If you create users in the Django admin interface and distribute the passwords to users, it should work.

**Comment by Pato05** [comment]
> If you create users in the Django admin interface and distribute the passwords to users, it should work.

Sorry for my ignorance, but how do I access the Django web interface? I found the little wrench icon when logged in as admin, but that brings me to some panel, but when adding users from there I can't choose their password, and it says "Confirmation link sent" (even without a webserver).

**Comment by Pato05** [comment]
Nevermind I found it, it was inside Tools > Django admin interface 🫥
